steps/terraform.yaml

```diff
@@ -109,12 +109,12 @@ steps:
       environmentServiceName: ${{ parameters.serviceConnection }}
       runAzLogin: true
       commandOptions: >
-        -out tfplan-${{ parameters.environment }}-${{ lower(replace(parameters.location, ' ', '')) }}-${{ parameters.component }}
```

Comment:
Added this because with a location based delineation, tf-cmt will not provide correct tags for this that exist in multiple regions (i.e. panorama)